README.rst

```diff
@@ -29,6 +29,6 @@ provider) where the strings can be translated.
 Compile translated strings by running `make compile_translations`. This will produce a .mo file for each .po in the repo.
 The .mo files are read by Django and are used to provide translations in the running application.
 
-Testing credentials-themes changes locally with your credentials repo
----------------------------------------------------------------------
-https://openedx.atlassian.net/wiki/spaces/SOL/pages/608698737/Testing+WL+themes+in+devstack
+Testing credentials-themes changes in devstack
+----------------------------------------------
+See instructions 'here <https://openedx.atlassian.net/wiki/spaces/SOL/pages/608698737/Testing+WL+themes+in+devstack>_'.
```

Comment:
Someone didn't fully internalize the a11y training. A better link might be something like "You can read [instructions for testing local credentials-themes] on the OpenEdx wiki." or something like that. But whatever. It's all good.